.circleci/config.yml

```diff
@@ -400,32 +475,79 @@ jobs:
               echo "Workflow not successful - ${created_workflow_status}"
               (exit -1) 
             fi
-            
+
             echo "Created workflow successful"
           environment:
             TMP_WORKSPACE: *tmp-workspace
             ENV_VARS_FILE: *env-vars-file
-  ios-sdk-size-report:
-    executor: mac-os-executor
+
+  run-ios-internal-sdk-e2e-tests:
+    executor: ubuntu-small-executor
 
     steps:
       - checkout
-      - ruby/install-deps
+      - attach_workspace:
+          at: *workspace
       - run:
-          name: Install Gem dependencies
+          name: Start automation pipeline to run E2E tests
           command: |
-            cd iosApp
-            bundle install
-      - run:
-          name: Configure GCloud App Distribution service
-          command: echo $GCLOUD_SERVICE_KEY > ${HOME}/gcloud-app-distribution-service-key.json
+            firebase_url=$(cat iosApp/fastlane/firebase_url.txt)
+            echo "Firebase URL: $firebase_url"
+            echo "export FIREBASE_URL=$firebase_url" >> $BASH_ENV
+            echo $FIREBASE_URL
+
+            CREATED_PIPELINE=$(curl --request POST \
+              --url https://circleci.com/api/v2/project/gh/twilio/twilio-verify-passkeys-tests/pipeline \
+              --header "Circle-Token: $CIRCLE_TOKEN" \
+              --header "Content-Type: application/json" \
+              --data '{
+                "branch":"appium-main",
+                "parameters":{
+                  "ios-app-url":"'"$FIREBASE_URL"'",
+                  "triggering-pipeline-id":"<< pipeline.id >>"
+                }
+              }' \
+              | jq -r '.id'
+            )
+            echo "<< pipeline.id >>"
+            echo "Created pipeline: $CREATED_PIPELINE"
+            ./scripts/add_env_variable_to_file.sh CREATED_PIPELINE $CREATED_PIPELINE $TMP_WORKSPACE $ENV_VARS_FILE
```

Comment:
Can you use a different env variable to store this pipeline? Android uses the same key, it could cause issues when both jobs are running in parallel at the same time. When you change it, can you also modify the Android to match the same naming convention you used for iOS?

Reply:
Fixed, thanks